### PR TITLE
Add sign up page

### PR DESCRIPTION
**Pull request by UtsavK55**

### CODE REVIEW

File: src/components/input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import {Alert, Text, TextInput, View} from 'react-native';
+import { styles } from './styles';
+
+const Input = ({label, invalid, style, textInputConfig}:InputProps) => {
+  const inputStyles = [styles.input];
+
+//   if (textInputConfig && textInputConfig?.multiline) {
```
Comment (02priyanshi) [inline src/components/input/Input.tsx]: remove comments

File: src/navigation/AuthNavigator.tsx
Comment (02priyanshi) [inline src/navigation/AuthNavigator.tsx:1]: create a constanst file for stack names and use constants for easily accesible

File: src/screens/signUp/SignUp.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      isValid: true,
+    },
+    confirmPass: {
+      value: '',
+      isValid: true,
+    },
+  });
+
+  const handleInputChange = (inputIdentifier: string, enteredValue: string) => {
+    setInputs(curInputs => {
```
Comment (02priyanshi) [inline src/screens/signUp/SignUp.tsx]: what is curInputs use proper name

File: src/screens/signUp/SignUp.tsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+          password: {value: curInputs.password.value, isValid: passwordIsValid},
+          confirmPass: {
+            value: curInputs.confirmPass.value,
+            isValid: confirmPassIsValid,
+          },
+        };
+      });
+      return;
+    }
+    console.log(signupData);
```
Comment (02priyanshi) [inline src/screens/signUp/SignUp.tsx]: remove console

File: src/components/button/Button.tsx
Comment (02priyanshi) [inline src/components/button/Button.tsx:1]: 1. how is this custom input if i need to send complete style as props
2.  Pressable itself work as a view comp don't need to wrap this inside view

**Review by 02priyanshi — CHANGES_REQUESTED** [review]

File: src/navigation/AuthNavigator.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import SignUp from '@src/screens/signUp/SignUp';
+import Login from '@src/screens/login/login';
```
Comment (shelly19gkmit) [inline src/navigation/AuthNavigator.tsx]: Maintain proper blank lines with the correct order of import statements-
import {createNativeStackNavigator} from '@react-navigation/native-stack';

import SignUp from '@src/screens/signUp/SignUp';
import Login from '@src/screens/login/login';

File: src/screens/signUp/SignUp.tsx [hunk 5]
```diff
@@ -0,0 +1,117 @@
+import {Button} from '@src/components/button';
+import {Input} from '@src/components/input';
+import {alphaRegex, emailRegex, passwordRegex} from '@src/constants/constant';
+import {useState} from 'react';
+import {SafeAreaView, Text} from 'react-native';
```
Comment (shelly19gkmit) [inline src/screens/signUp/SignUp.tsx]: fix import orders-

import {useState} from 'react';
import {SafeAreaView, Text} from 'react-native';

import {Button} from '@src/components/button';
import {Input} from '@src/components/input';
import {alphaRegex, emailRegex, passwordRegex} from '@src/constants/constant';

**Review by 02priyanshi — APPROVED** [review]

**Pull request merged**